### ISSUE TITLE
Anyone aware of L4-L7 resource deployment (Service Graph, Devices, PBR)?

### ISSUE THREAD

**Issue by rypatterson** [CiscoDevNet/terraform-provider-aci#86]
Deploying a new ACI environment and so far I've been able to deploy everything using the TF ACI Provider with the exception of L4-L7 services. Any assistance is greatly appreciated. 

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s) + ACI Class(es):

* aci_XXXXX + fv:XXXX

### APIC version and APIC Platform

* V x.x.x and on-prem/cloud-aws/cloud-azure/all.

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by nkatarmal-crest** [comment]
@rypatterson Can you please add the class names which you are looking for? I can help with resource docs or can add new resources if we don't have in our bag.

**Comment by rypatterson** [comment]
Hello @nkatarmal-crest 
Thank you very much for your support. I've attached an example JSON for a PBR policy I've configured within ACI. If this isn't helpful or you need additional information please let me know. 

[svcRedirectPol-kcs-vl-916-pbr.txt](https://github.com/terraform-providers/terraform-provider-aci/files/4957619/svcRedirectPol-kcs-vl-916-pbr.txt)

**Comment by nkatarmal-crest** [comment]
@rypatterson We don't have resources for this classes yet in the provider. We need to add new resources for them.

**Comment by nkatarmal-crest** [comment]
This issue has been fixed in the version 0.5.0 of the provider. Please try that and let me know if we can close this issue or not.

**Comment by FrederikSuijs** [comment]
@nkatarmal-crest what resources are new exactly?
I have tried looking for vnsLIf,vnsCDev,vndAbsGraph,vnsAbsConnection... as they are called in the MO, but couldn't find any.
